Sources/Brave/WebFilters/AdBlockEngineManager.swift

```diff
@@ -187,38 +187,36 @@ public actor AdBlockEngineManager: Sendable {
 extension AdBlockEngineManager {
   /// A method that logs info on the given resources
   fileprivate func debug(compiledResults: [ResourceWithVersion: Result<Void, Error>]) {
-    let resourcesString = compiledResults.sorted(by: { $0.key.order < $1.key.order })
-      .map { (resourceWithVersion, compileResult) -> String in
+    log.debug("Loaded \(compiledResults.count, privacy: .public) (total) engine resources:")
```

Comment:
Does this need to be public?

Reply:
Fixed

Reply:
Sorry I didn't mean the extension, I meant the log itself (`privacy: .public`), this code is only compiled in Debug anyways, so it doesn't need to be explicitly public

Reply:
Ahh right. Fixed

Reply:
You only fixed this for the log below? But not this one? `privacy: .private` is redundant as well btw, its the default